MekHQ/src/mekhq/campaign/mission/ScenarioForceTemplate.java

```diff
@@ -504,6 +506,18 @@ public boolean isEnemyBotForce() {
                 getForceAlignment() == ForceAlignment.Third.ordinal(); 
     }
     
+    /**
+     * Convenience function that returns the displayable name of the selected unit type.
+     * @return
+     */
+    public String getAllowedUnitTypeName() {
+        if(getAllowedUnitType() > UnitType.SIZE || getAllowedUnitType() < 0) {
+            return SPECIAL_UNIT_TYPES.get(getAllowedUnitType());
+        } else {
+            return UnitType.getTypeDisplayableName(getAllowedUnitType());
+        }
+    }
+    
     /**
      * Attempt to deserialize an instance of a ScenarioTemplate from the passed-in XML Node
      * @param inputFile The source file
```

Comment:
Not part of your current changes, but the documented parameter does not match the actual parameter. The same is true of the deserialize method a few lines below this, which does not appear in this changelog.